isis/src/control/apps/cnetbin2pvl/cnetbin2pvl.h

```diff
@@ -0,0 +1,13 @@
+#ifndef cnetbin2pvl_h
+#define cnetbin2pvl_h
+
+#include "UserInterface.h"
+#include "ControlNet.h"
+#include "Progress.h"
+
+namespace Isis {
+  extern void cnetbin2pvl(UserInterface &ui);
+  extern void cnetbin2pvl(ControlNet &net, Progress &progress, UserInterface &ui);
```

Comment:
Progress should be a pointer that defaults to nullptr so that it's not used when this is called outside of the app.